docs/whats-news/2022/2022-08.md

```diff
@@ -0,0 +1,39 @@
+---
+id: 2022-08
+title: What's Newsletter August 2022
+slug: /2022-08
+---
+
+# What's New in Nimbus August 2022
+
+## Welcome
+
+The Nimbus team will have a **demo booth at #moz-aloha** All-Hands at the end of September! Come find us, chat about experimentation, and maybe even grab a sticker!
+
+## Exciting Changes
+
+### Mobile First Run
```

Comment:
I think we should add here little info what does mobile-first run feature mean

Reply:
Sounds like a good idea! I added a sentence about it to the latest commit, let me know if you have any suggestions to improve wording/clarity.